dts/bindings/adc/st,stm32-adc.yaml

```diff
@@ -21,6 +21,21 @@ properties:
     "#io-channel-cells":
       const: 1
 
+    st,prescaler:
+      required: false
+      type: array
+      description: |
+        The ADC Clock specifier at the input of the peripheral.
+        Depending on the soc serie, it is ASYNChronous specific clock
+        or SYNChronous derived from the AHB or APB.
```

Comment:
```suggestion
        On some series, it could be either ASYNChronous (derived from the domain clock)
        or SYNChronous (derived from the AHB or APB).
        In each case, a specific prescaler can be applied to reduce the input frequency.
```

Reply:
changed